apis/runtime/v1alpha1/api.proto

```diff
@@ -111,8 +111,10 @@ message HugepageLimit {
 message ContainerMetadata {
   // Name of the container. Same as the container name in the PodSpec.
   string name = 1;
+  // Id of the container.
+  string id = 2;
   // Attempt number of creating the container. Default: 0.
-  uint32 attempt = 2;
+  uint32 attempt = 3;
```

Comment:
Should we keep the number to keep compatibility?

Reply:
Done